projects/commudle-admin/src/app/feature-modules/sys-admin/components/admin-badges/admin-badges-assign/admin-badges-assign.component.scss

```diff
@@ -0,0 +1,44 @@
+@import '/projects/commudle-admin/src/assets/styles/variables';
```

Comment:
Why was this css file formatted with 4 spaces instead of 2 spaces?

Reply:
done with 2 spaces